### DISCUSSION TITLE
x86 to x64 upgrade & RelatedBundle

### DISCUSSION THREAD

**Discussion by WacomKev** [wixtoolset/issues#7412] [Questions]
I'm building a new, 64-bit version of an existing MBA installer. I'd added a RelatedBundle element to remove the old, 32-bit version. However, it appears that removal of the old version occurs _after_ the new version has been installed. This is causing some issues. Is there a way to have the old version removed earlier in the process?

Thanks

**Comment by rseanhall** [comment]
No, that's not possible today. What issues are you running into?

**Comment by WacomKev** [comment]
It turns out the issue isn't my upgrade from 32- to 64-bit. 
The problem is a set of 3rd party MSIs. We recently upgraded to a new version of that product and the new version has a new, different set of MSIs. So our new installer installs the new MSIs and then the old MSIs get uninstalled (by our cached old installer). It would appear that leaves the 3rd party product in an incomplete or unstable state and it doesn't work correctly.
I guess if Bundle had the equivalent of MajorUpgrade Schedule="afterInstallValidate" it wouldn't be an issue. As it doesn't a I seem to be a bit stuck

**Reply by rseanhall** [reply]
I believe https://github.com/wixtoolset/issues/issues/4858 is tracking this scenario.
